srcpkgs/go1.22-bootstrap/template

```diff
@@ -0,0 +1,51 @@
+# Template file for 'go1.22-bootstrap'
+pkgname=go1.22-bootstrap
+version=1.22.0
+revision=1
+archs="x86_64* i686* armv[67]l* aarch64* ppc64le*"
+short_desc="Go 1.22 (bootstrap compiler)"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="BSD-3-Clause"
+homepage="https://golang.org"
+nostrip=yes
+noverifyrdeps=yes
+nocross=yes
+lib32disabled=yes
+repository=bootstrap
+
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	depends+=" gcompat"
+fi
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*)
+		_dist_arch="amd64"
+		checksum="f6c8a87aa03b92c4b0bf3d558e28ea03006eb29db78917daec5cfb6ec1046265"
+		;;
+	i686*)
+		_dist_arch="386"
+		checksum="1e209c4abde069067ac9afb341c8003db6a210f8173c77777f02d3a524313da3"
+		;;
+	arm*)
+		_dist_arch="armv6l"
+		checksum="0525f92f79df7ed5877147bce7b955f159f3962711b69faac66bc7121d36dcc4"
+		;;
+	aarch64*)
+		_dist_arch="arm64"
+		checksum="6a63fef0e050146f275bf02a0896badfe77c11b6f05499bb647e7bd613a45a10"
+		;;
+	ppc64le*)
+		_dist_arch="ppc64le"
+		checksum="0e57f421df9449066f00155ce98a5be93744b3d81b00ee4c2c9b511be2a31d93"
+		;;
+esac
+
+distfiles="https://dl.google.com/go/go${version}.linux-${_dist_arch}.tar.gz"
+
+do_install() {
+	vmkdir usr/lib/go${version}
+	vcopy bin usr/lib/go${version}
+	vcopy src usr/lib/go${version}
+	vcopy pkg usr/lib/go${version}
```

Comment:
does it make sense to make this `go1.22` (`go${version%.*}`) so the main template doesn't have to change if the bootstrap is changed?

Reply:
Yes, but there's also no good reason to update the bootstrap I think (unless a serious miscompilation is detected).